pkg/controller/daemon/daemoncontroller.go

```diff
@@ -1142,33 +1142,21 @@ func (dsc *DaemonSetsController) nodeShouldRunDaemonPod(node *v1.Node, ds *exten
 			dsc.eventRecorder.Eventf(ds, v1.EventTypeWarning, FailedPlacementReason, "failed to place pod on %q: %s", node.ObjectMeta.Name, reason.Error())
 			shouldSchedule = false
 		case *predicates.PredicateFailureError:
-			var emitEvent bool
 			switch reason {
-			// unintentional predicates reasons need to be fired out to event.
+			// unexpected
 			case
 				predicates.ErrDiskConflict,
 				predicates.ErrVolumeZoneConflict,
 				predicates.ErrMaxVolumeCountExceeded,
 				predicates.ErrNodeUnderMemoryPressure,
```

Comment:
Yes, the DS do not see them, so I want to treat them as **unexpected** predicate errors, comments?